cloudformation/stack.yaml

```diff
@@ -3797,7 +3798,7 @@ Outputs:
 
   JanusExternalPort:
     Description: Janus External Port [reticulum/janus/janus_port]
-    Value: !FindInMap [ServicesMeta, DialogExternal, Port]
+    Value: !If [HasStreamingServers, !FindInMap [ServicesMeta, DialogExternal, StreamPort], !FindInMap [ServicesMeta, DialogExternal, AppPort] ]
```

Comment:
I noticed that `HasStreamingServers` is false when the stack is set to Offline. I don't think it will break anything, but it's a bit surprising, so perhaps another thing to keep in mind if we run into any quirks later.

Reply:
I think that's correct. There's neither streaming nor app servers when Offline mode is on since there's no EC2 instances up.